keel-echo/src/test/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListenerTests.kt

```diff
@@ -91,7 +91,7 @@ class UnhealthyNotificationListenerTests : JUnit5Minutests {
           subject.onResourceHealthEvent(event)
           verify(exactly = 1) {
             publisher.publishEvent(
-              NotificationEvent(RESOURCE, r.id, UNHEALTHY_RESOURCE, subject.message(event, Duration.ofMinutes(6)))
+              UnhealthyNotification(RESOURCE, r.id, UNHEALTHY_RESOURCE, subject.message(event, Duration.ofMinutes(6)))
```

Comment:
Passing the `UNHEALTHY_RESOURCE` type is redundant with the constructor here, right?

Reply:
removed it!